packages/run-protocol/test/vaultFactory/test-vaultFactory.js

```diff
@@ -2879,3 +2880,48 @@ test('manager notifiers', async t => {
     totalDebt: { value: 0n },
   });
 });
+
+test('governance publisher', async t => {
+  const { aeth } = t.context;
+  t.context.loanTiming = {
+    chargingPeriod: 2n,
+    recordingPeriod: 10n,
+  };
+
+  const services = await setupServices(
+    t,
+    [500n, 15n],
+    aeth.make(900n),
+    undefined,
+    undefined,
+    500n,
+  );
+  const { lender } = services.vaultFactory;
+  const directorGovNotifier = makeNotifierFromAsyncIterable(
+    E(lender).getElectorateSubscription(),
+  );
+  let {
+    value: { current },
+  } = await directorGovNotifier.getUpdateSince();
+  // can't deepEqual because of non-literal objects
+  t.is(current.Electorate.type, 'invitation');
+  t.is(current.LiquidationInstall.type, 'installation');
+  t.is(current.LiquidationTerms.type, 'unknown');
+  t.is(current.MinInitialDebt.type, 'amount');
+  t.is(current.ShortfallInvitation.type, 'invitation');
+
+  const managerGovNotifier = makeNotifierFromAsyncIterable(
+    E(lender).getSubscription({
+      collateralBrand: aeth.brand,
+    }),
+  );
+  ({
+    value: { current },
+  } = await managerGovNotifier.getUpdateSince());
+  // can't deepEqual because of non-literal objects
```

Comment:
Can we find a test where governance changes a value (`packages/run-protocol/test/swingsetTests/governance/test-governance.js`, e.g.) and show that the values from `getUpdateSince()` before and after are different?

Reply:
I'm reading this as: make sure that when governance changes a value, it changes in the subscription. I think dba994f884bf48c4ed5a2f14e5dd6aef96e2da73 covers this but lemme know.